vcd/resource_vcd_external_network.go

```diff
@@ -0,0 +1,381 @@
+package vcd
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+func resourceVcdExternalNetwork() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceVcdExternalNetworkCreate,
+		Delete: resourceVcdExternalNetworkDelete,
+		Read:   resourceVcdExternalNetworkRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+			"ip_scope": &schema.Schema{
```

Comment:
Is this term IP scope coming from vCD API? It's the first time we expose it for the user.

Reply:
No, it's general name - if you have better, please provide.